src/main/steps/application-within-proceedings/document-upload/content.ts

```diff
@@ -0,0 +1,150 @@
+  uploadYourApplication:
+    'Upload your application to the case. If you are uploading a paper copy of the application, make sure this has been scanned in clearly, and saved in a suitable file format such  as PDF.',
+  uploadDescription: 'Take a picture of a document on your phone and upload it',
+  uploadRequirements: [
+    'Place your document on a flat service in a well-lit room. Use a flash if you need to.',
+    'Take a picture of the whole document. You should be able to see its edges.',
+    'Check you can read all the writing, including the handwriting.',
+    'Email or send the photo or scan to the device you are using now.',
+    'Upload it here.',
+  ],
+  onlyContinue: 'Continue',
+  cancel: 'Cancel',
+  uploadButton: 'Upload file',
+  remove: 'Remove',
+  errorText: 'Error:',
+  noFilesText: 'No files uploaded',
+  errors: {
+    document: {
+      required: 'Upload your {applicationType} application form',
+      fileFormat: 'The file you uploaded is in the wrong format. Upload your file again in the correct format',
+      fileSize: 'The file you uploaded is too large. Maximum file size allowed is 20MB',
+    },
+  },
+};
+
+export const cy: typeof en = {
+  title: 'Upload your application (welsh)',
+  uploadYourApplication:
+    'Upload your application to the case. If you are uploading a paper copy of the application, make sure this has been scanned in clearly, and saved in a suitable file format such  as PDF. (welsh)',
+  uploadDescription: 'Take a picture of a document on your phone and upload it (welsh)',
+  uploadRequirements: [
+    'Rhowch eich dogfen ar rywbeth gwastad mewn ystafell sydd â digon o olau. Defnyddiwch fflach y camera os bydd angen.',
+    "Tynnwch lun o’r ddogfen gyfan. Dylech allu gweld corneli'r ddogfen.",
+    'Gwiriwch eich bod yn gallu gweld yr ysgrifen i gyd, gan gynnwys y llawysgrifen.',
+    'Anfonwch y llun trwy e-bost neu sganiwch y ddogfen i’r ddyfais rydych yn ei defnyddio nawr.',
+    'Llwythwch y ffeil yma.',
+  ],
+  onlyContinue: 'Parhau',
+  cancel: 'Canslo',
+  uploadButton: 'Llwytho ffeil',
+  remove: 'Dileu',
+  errorText: 'Error: (welsh)',
+  noFilesText: 'No files uploaded (welsh)',
+  errors: {
+    document: {
+      required: 'Upload your {applicationType} application form (welsh)',
+      fileFormat: "Mae'r ffeil a lwythwyd gennych yn y fformat anghywir. Llwythwch eich ffeil eto yn y fformat cywir.",
+      fileSize: "Mae'r ffeil yr ydych wedi ei llwytho yn rhy fawr",
+    },
+  },
+};
+
+const languages = {
+  en,
+  cy,
+};
+
+export const form: FormContent = {
+  fields: {
+    awp_documentUpload: {
+      type: 'hidden',
+      label: l => l.uploadFiles,
+      labelHidden: true,
+      value: 'true',
+    },
+  },
+  onlyContinue: {
+    text: l => l.onlyContinue,
+  },
+  link: {
+    classes: 'govuk-!-margin-left-3',
+    href: '/application-within-proceedings/list-of-applications/1',
+    text: l => l.cancel,
+  },
+};
+
+const removeDocument = async (req, removeId) => {
```

Comment:
This method is not required. please write the logic for removing the doc inside generate content.